Tests/Functional/config/bc_automatic_controller_injections.yml

```diff
@@ -0,0 +1,12 @@
+imports:
+    - { resource: default.yml }
+    
+jms_di_extra:
+    automatic_controller_injections:
+        properties:
+            context: "@security.context"
```

Comment:
You can't use the SecurityContext in SF3.0, it's removed (deprecated since 2.6)
You should remove it and bump the SF version requirements to 2.6, it's the lowest BC that we can achieve from 3.0